Makefile

```diff
@@ -36,7 +36,7 @@ docker_stop_spanner:
 	docker-compose -f docker-compose.spanner.yaml down
 
 run:
-	RUST_LOG=debug RUST_BACKTRACE=full cargo run -- --config config/local.toml --features tokenserver_test_mode
+	RUST_LOG=debug RUST_BACKTRACE=full cargo run --features tokenserver_test_mode -- --config config/local.toml
```

Comment:
This was a bug: https://doc.rust-lang.org/cargo/commands/cargo-run.html#description